docker/django_entrypoint.sh

```diff
@@ -13,6 +13,7 @@ python manage.py migrate
 python manage.py collectstatic --no-input
 
 python manage.py validate_target_json
+python manage.py delete_mediafiles
```

Comment:
Just in case I don't find this later... I'm assuming that this deletes mediafiles based on their timestamps/time of operation/etc.

If so, perhaps renaming to delete_outdated_mediafiles?